app/services/users_service.rb

```diff
@@ -17,6 +17,8 @@ def login(email, password)
   end
 
   def register(email, password, organization_name)
+    return result.not_allowed_failure!(code: 'signup is disabled') if ENV.fetch('LAGO_SIGNUP_DISABLED', 'false') == 'true'
```

Comment:
Layout/LineLength: Line is too long. [122/120] (https://rubystyle.guide#max-line-length)